src/components/Chart.js

```diff
@@ -6,6 +6,9 @@ import { h } from 'react-hyperscript-helpers';
 
 highchartsAccessibility(Highcharts);
 highchartsExporting(Highcharts);
+// We do not (and should not) transform user input into links or other HTML using Highcharts;
+// this is needed to permit use of internally constructed links in a Highcharts formatter
+Highcharts.AST.bypassHTMLFiltering = true;
```

Comment:
This is necessary because we are now constructing links to workspaces and using them in the Highcharts label formatter. The links are variables which come from a call to the existing `Nav` utility, and Highcharts will warn about any `href` attribute whose link does not explicitly start with a "safe" built-in or user-defined prefix (e.g. `https`).

What Highcharts has to say about this feature ([link](https://api.highcharts.com/class-reference/Highcharts.AST#.bypassHTMLFiltering)):
> The `bypassHTMLFiltering` setting is intended only for those cases where allow-listing is not practical, and the chart configuration already comes from a secure source.

Allow-listing is not practical because we can't provide an explicit prefix to the `Nav` links to allow-list. The configuration comes from a secure source (our own utilities for getting links to Terra pages).